ui/app/templates/components/form-field.hbs

```diff
@@ -88,6 +88,16 @@
     inputValue=(get model valuePath)
     onChange=(action (action "setAndBroadcast" valuePath))
   }}
+{{else if (eq attr.options.label 'Password')}}
```

Comment:
I don't think we should rely on the label to render the right thing here. We could probably add `sensitive: true` to the model attributes and use that to determine if we render the `masked-input` component.